docs/what-is-hipFORT.rst

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
This topic should be removed. 
The single sentence could be blended into the landing page.